src/Mod/Start/StartPage/LoadNew.py

```diff
@@ -26,32 +26,28 @@
 # template will be given before this script is run
 template_name = str(template)
 
-match template_name:
-    case "empty_file":
+if template_name == "empty_file":
         FreeCADGui.runCommand('Std_New')
         StartPage.postStart()
-    case "import_file":
+elif template_name == "import_file":
```

Comment:
Can you please fix the indentation of this block, now that it's just conditionals and not a match/case?

Reply:
Done